### PR TITLE
add init of BatchNorm2D

### PR DESCRIPTION
**Pull request by WenmuZhou**
### PR types
New features
### PR changes
APIs

### CODE REVIEW

File: padiff/special_init/init_BatchNorm2D.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+from .special_init_pool import global_special_init_pool as init_pool
+
+
+@init_pool.register(paddle_name="BatchNorm2D", torch_name="BatchNorm2d")  # 此处填写模型的类名
+def init_BatchNorm2D(layer, module):
+    param_dict = {}
+    for name, param in module.state_dict().items():
+        name = name.replace("running_var", "_variance").replace("running_mean", "_mean")
+        param_dict[name] = paddle.to_tensor(param.cpu().detach().numpy())
```
Comment (feifei-111) [inline padiff/special_init/init_BatchNorm2D.py:25]: 注意一下device呢，可以用paddle.get_device

**Review by feifei-111 — APPROVED** [review]
LGTM
